azure-pipelines.yaml

```diff
@@ -10,10 +10,8 @@ pool:
 
 strategy:
   matrix:
-    'PHP 7.4':
-      PHP_VERSION: 7.4
-    'PHP 8.0':
-      PHP_VERSION: 8.0
+    'PHP 8.2':
+      PHP_VERSION: 8.2
```

Comment:
8.1 should be here as it's the oldest supported version. I would consider 8.2 optional (there are not that many BC between 8.1 and 8.2 as there was between 7.4 and 8.0